Cargo.toml

```diff
@@ -11,7 +11,7 @@ categories = ["network-programming", "cryptography"]
 
```

Comment:
Do you want to go ahead and bump the `Cargo.toml` version alongside this? We can prepare a release right out of this branch.